modules/logging-release-notes-5-8-0.adoc

```diff
@@ -6,13 +6,17 @@ This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenS
 
 [id="logging-release-notes-5-8-0-deprecation-notice"]
 == Deprecation notice
+
 In Logging 5.8, Elasticsearch, Fluentd, and Kibana are deprecated and are planned to be removed in Logging 6.0, which is expected to be shipped alongside a future release of {product-title}. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the {clo} and LokiStack provided by the {loki-op} are the preferred Operators for log collection and storage. We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
 
 [id="logging-release-notes-5-8-0-enhancements"]
 == Enhancements
 
 [id="logging-release-notes-5-8-0-log-collection"]
 === Log Collection
+
+* With this update, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default. You must manually create a `LogFileMetricExporter` CR to generate metrics from the logs produced by running containers. If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*. (link:https://issues.redhat.com/browse/LOG-3819[LOG-3819])
```

Comment:
With this update, the `LogFileMetricExporter` is no longer deployed with the collector by default